### PR TITLE
Fix test suite

### PR DESCRIPTION
**Pull request by tacataca**
* Running the rake and rake test was missing the services, job and mailer directories
* Fix some service tests that were out of sync with changes made to the app

### CODE REVIEW

File: lib/tasks/tests.rake [hunk 1]
```diff
@@ -17,7 +17,7 @@ task :default do
 end
 
 Rake::TestTask.new('test:unit') do |t|
-  t.pattern = 'test/{controllers,lib,models,interactors}/**/*_test.rb'
+  t.pattern = 'test/{controllers,jobs,lib,mailers,models,services}/**/*_test.rb'
```
Comment (sheldond) [inline lib/tasks/tests.rake]: Would it be easier and more future-proof to just run all tests in all dirs, except the integration dir?

File: app/services/stay/list_child_costs.rb [hunk 2]
```diff
@@ -46,7 +46,7 @@ def child_cost(child)
     if child.age >= 5
       stay.needs_bed? ? :child : :child_meal
     else
-      stay.needs_bed? ? :infant : nil
+      stay.needs_bed? ? :infant : :child_meal
```
Comment (sheldond) [inline app/services/stay/list_child_costs.rb]: I'm not too familiar with the business rules here, but it looks like we're changing the cost from, I guess, no cost (nil) to something that has a cost (child_meal). I just wanted to make sure that this change is inline with desired business rules. What do you think?

Comment (tacataca) [reply app/services/stay/list_child_costs.rb]: Good call on this.
Confirmed with client that it is supposed to be nil, thanks so much 🎖

File: lib/tasks/tests.rake [hunk 3]
```diff
@@ -17,7 +17,9 @@ task :default do
 end
 
 Rake::TestTask.new('test:unit') do |t|
-  t.pattern = 'test/{controllers,lib,models,interactors}/**/*_test.rb'
+  t.test_files = FileList['test/**/*_test.rb'].exclude(
+    'test/integration/**/*_test.rb'
+  )
```
Comment (sheldond) [inline lib/tasks/tests.rake]: Thanks for figuring this out! I didn't know about `FileList`!

**Pull request merged**